include/Nazara/VulkanRenderer/Wrapper/Surface.inl

```diff
@@ -6,6 +6,7 @@
 #include <Nazara/Core/Error.hpp>
 #include <Nazara/VulkanRenderer/Utils.hpp>
 #include <Nazara/VulkanRenderer/Debug.hpp>
+#include <vulkan/vulkan_metal.h>
```

Comment:
Useless as it's already part of Surface.hpp

Reply:
Removed, indeed.